python/example_code/ec2/scenario_get_started_instances.py

```diff
@@ -0,0 +1,261 @@
+        self.ssm_client = ssm_client
+
+    @demo_func
+    def create_and_list_key_pairs(self):
+        """
+        1. Creates an RSA key pair and saves its private key data as a .pem file in secure
+           temporary storage. The private key data is deleted after the example completes.
+        2. Lists the first five key pairs for the current account.
+        """
+        print("Let's create an RSA key pair that you can be use to securely connect to "
+              "your Amazon EC2 instance.")
+        key_name = q.ask("Enter a unique name for your key: ", q.non_empty)
+        self.key_wrapper.create(key_name)
+        print(f"Created a key pair {self.key_wrapper.key_pair.key_name} and saved the "
+              f"private key to {self.key_wrapper.key_file_path}.\n")
+        if q.ask("Do you want to list some of your key pairs? (y/n) ", q.is_yesno):
+            self.key_wrapper.list(5)
+
+    @demo_func
+    def create_security_group(self):
+        """
+        1. Creates a security group for the default VPC.
+        2. Adds an inbound rule to allow SSH. The SSH rule allows only
+           inbound traffic from the current computer’s public IPv4 address.
+        3. Displays information about the security group.
+
+        This function uses 'http://checkip.amazonaws.com' to get the current public IP
+        address of the computer that is running the example. This method works in most
+        cases. However, depending on how your computer connects to the internet, you
+        may have to manually add your public IP address to the security group by using
+        the AWS Management Console.
+        """
+        print("Let's create a security group to manage access to your instance.")
+        sg_name = q.ask("Enter a unique name for your security group: ", q.non_empty)
+        security_group = self.sg_wrapper.create(
+            sg_name, "Security group for example: get started with instances.")
+        print(f"Created security group {security_group.group_name} in your default "
+              f"VPC {security_group.vpc_id}.\n")
+
+        ip_response = urllib.request.urlopen('http://checkip.amazonaws.com')
+        current_ip_address = ip_response.read().decode('utf-8').strip()
+        print("Let's add a rule to allow SSH only from your current IP address.")
+        print(f"Your public IP address is {current_ip_address}.")
+        q.ask("Press Enter to add this rule to your security group.")
+        response = self.sg_wrapper.authorize_ingress(current_ip_address)
+        if response['Return']:
+            print("Security group rules updated.")
+        else:
+            print("Couldn't update security group rules.")
+        self.sg_wrapper.describe()
+
+    @demo_func
+    def create_instance(self):
+        """
+        1. Gets a list of Amazon Linux 2 AMIs from Systems Manager. Specifying the
+           '/aws/service/ami-amazon-linux-latest' path ensures that only the latest AMIs
```

Comment:
ensures that only the latest AMIs are returned. --> returns only the latest AMIs.

Reply:
fixed